py-server/chat_client.py

```diff
@@ -4,14 +4,15 @@
 import sys
 import textwrap
 import click
-from main import chat_parameterized
-from agents.registry import AgentRegistry
+from main import chat_parameterized, agent_registry
```

Comment:
we shouldn't import from main - please add a TODO to move chat_parametrized to a separate file and instantiate an `AgentRegistry` instead of importing the global variable from main.